src/js/components/awardv2/shared/overview/RelatedAwards.jsx

```diff
@@ -42,47 +43,67 @@ export default class RelatedAwards extends React.Component {
+        this.props.jumpToSubAwardHistoryTable('subaward');
+        this.props.jumpToSection('award-history');
+    }
 
     referencedAwardCounts() {
-        const { counts } = this.props;
+        const { counts, overview } = this.props;
         if (!counts) return null;
-        const childData = [
-            {
-                count: formatNumber(counts.child_awards),
-                name: 'Child Award',
-                funcName: 'ChildAwards',
-                glossary: 'contract'
-            },
-            {
-                count: formatNumber(counts.child_idvs),
-                name: 'Child IDV',
-                funcName: 'ChildIDVs',
-                glossary: 'IDV'
-            },
-            {
-                count: formatNumber(counts.grandchild_awards),
-                name: 'Grandchild Award',
-                funcName: 'GrandchildAwards',
-                glossary: 'award'
-            }
-        ];
-
+        let childData = [];
+        if (overview.category === 'idv') {
+            childData = [
+                {
+                    count: formatNumber(counts.child_awards),
+                    name: 'Child Award',
+                    funcName: 'jumpToReferencedAwardsTableChildAwardsTab',
+                    glossary: 'contract',
+                    postText: counts.child_awards === 1 ? 'Order' : 'Orders'
+                },
+                {
+                    count: formatNumber(counts.child_idvs),
+                    name: 'Child IDV',
+                    funcName: 'jumpToReferencedAwardsTableChildIDVsTab',
+                    glossary: 'IDV',
+                    postText: counts.child_idvs === 1 ? 'Order' : 'Orders'
+                },
+                {
+                    count: formatNumber(counts.grandchild_awards),
+                    name: 'Grandchild Award',
+                    funcName: 'jumpToReferencedAwardsTableGrandchildAwardsTab',
+                    glossary: 'award',
+                    postText: counts.grandchild_awards === 1 ? 'Order' : 'Orders'
+                }
+            ];
+        }
+        else {
+            childData = [
+                {
+                    count: formatNumber(counts.subawardCount),
+                    name: 'Sub-Awards',
+                    funcName: 'jumpToAwardHistoryTableSubAwardsTab',
+                    glossary: 'contract',
+                    postText: ''
+                }
+            ];
+        }
         return (
             <div className="related-awards__label related-awards__label_count">
                 <div className="related-awards__counts">
                     {map(childData, (data) => (
                         <button
                             key={`${data.glossary}count`}
                             className="award-viz__button"
-                            onClick={this[`jumpToReferencedAwardsTable${data.funcName}Tab`]}>
+                            onClick={this[`${data.funcName}`]}>
```

Comment:
This is an anti-pattern because if we wanted to see all the callers of `jumpToAwardHistoryTableSubAwardsTab` or one of the other `childData[i].funcName`, we couldn't rely on a basic global cntrl + f search.

One solution is to use a map inside the render where we map `funcName` to the class method. So then, we can see where the map is used to identify the caller.

In this context though, I don't think it's a big deal. I'm sure we're doing this elsewhere and it's just not a good pattern to use uncritically.

Reply:
Again, defer to your judgment as to whether or not to clean this up now.